packages/documentation/package.json

```diff
@@ -36,7 +37,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "if [ -z \"$GITHUB_API_KEY\" ]; then echo \"No API key, skipping doc build\"; else gatsby build; fi",
+    "build": "if [ -z \"$GITHUB_API_KEY\" ]; then echo \"No API key, skipping doc build\"; else gatsby build --prefix-paths && gh-pages -d public -e ../..; fi",
```

Comment:
I verified that it pushes the build locally to the gh-pages [branch](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/gh-pages/). I don't have access to the settings for this project to turn on https://department-of-veterans-affairs.github.io/veteran-facing-services-tools

Reply:
I can get this turned on when the DSVA admin person is back online this afternoon.

Reply:
Looks good!